libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/model/RouteSetValue.kt

```diff
@@ -22,10 +20,8 @@ class RouteSetValue internal constructor(
     val trafficLineExpressionProvider: RouteLineExpressionProvider?,
     val routeLineExpression: Expression,
     val casingLineExpression: Expression,
-    val altRoute1TrafficExpressionProvider: RouteLineExpressionProvider?,
-    val altRoute2TrafficExpressionProvider: RouteLineExpressionProvider?,
-    val alternativeRoute1Source: FeatureCollection,
-    val alternativeRoute2Source: FeatureCollection,
+    val alternativeRoutesTrafficExpressionProviders: List<RouteLineExpressionProvider?>,
+    val alternativeRoutesSources: List<FeatureCollection>,
```

Comment:
Same as above, can this be a list of objects that contain the sourceID and the FeatureCollection so the view class doesn't have to know how to match them up?